.github/workflows/cypress.yml

```diff
@@ -20,13 +30,13 @@ jobs:
         env:
           HUSKY_SKIP_INSTALL: true
 
-      - name: run Cypress
-        run: npx run-p --race start cypress:run
+      - name: run Cypress CI
+        run: npx run-p --race start cypress:ci
         env:
           CI: true
+          TERM: xterm-256color
           CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
           DEBUG: commit-info,cypress:server:record
           # https://docs.cypress.io/guides/guides/continuous-integration.html#Environment-variables
           COMMIT_INFO_BRANCH: ${{ github.head_ref }}
-          COMMIT_INFO_AUTHOR: ${{ github.event.sender.login }}
-          COMMIT_INFO_SHA: ${{ github.event.after }}
```

Comment:
What was the reason for removing these environment variables? Since merging i'm getting errors like the following from cypress: https://github.com/jshjohnson/Choices/runs/283664081#step:5:0 :)

Reply:
@jshjohnson I've added them just to help `commitinfo`, but it was able to find the same information in the previous run. You may revert that, it doesn't hurt to have them here.

Reply:
But errors seems unrelated

Reply:
yeah i think i know where the problem is - we're not passing a value to `--group` in the `cypress:ci` script

Reply:
@jshjohnson github.event.after will not be available on `push` to master, so, if you would like to reuse the same action you may better remove COMMIT_INFO_SHA completely here